commit b507796679529b14c99e8937870561cd8eebebb9
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Tue May 28 12:13:02 2013 +0100

    Add the known marketing names for the performance Haswell parts
    
    Start filling in the names for the parts that have been announced, the
    Iris branded Haswell GT3 parts.
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/src/intel_module.c b/src/intel_module.c
index 1e402f0..8b3b196 100644
--- a/src/intel_module.c
+++ b/src/intel_module.c
@@ -192,34 +192,34 @@ static const SymTabRec intel_chipsets[] = {
 	{PCI_CHIP_HASWELL_SDV_E_GT3,		"Haswell SDV (GT3)" },
 	{PCI_CHIP_HASWELL_ULT_D_GT1,		"Haswell ULT Desktop (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_D_GT2,		"Haswell ULT Desktop (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_D_GT3,		"Haswell ULT Desktop (GT3)" },
+	{PCI_CHIP_HASWELL_ULT_D_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_ULT_M_GT1,		"Haswell ULT Mobile (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_M_GT2,		"Haswell ULT Mobile (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_M_GT3,		"Haswell ULT Mobile (GT3)" },
+	{PCI_CHIP_HASWELL_ULT_M_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_ULT_S_GT1,		"Haswell ULT Server (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_S_GT2,		"Haswell ULT Server (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_S_GT3,		"Haswell ULT Server (GT3)" },
+	{PCI_CHIP_HASWELL_ULT_S_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_ULT_B_GT1,		"Haswell ULT (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_B_GT2,		"Haswell ULT (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_B_GT3,		"Haswell ULT (GT3)" },
+	{PCI_CHIP_HASWELL_ULT_B_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_ULT_E_GT1,		"Haswell ULT (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_E_GT2,		"Haswell ULT (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_E_GT3,		"Haswell ULT (GT3)" },
+	{PCI_CHIP_HASWELL_ULT_E_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_CRW_D_GT1,		"Haswell CRW Desktop (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_D_GT2,		"Haswell CRW Desktop (GT2)" },
-	{PCI_CHIP_HASWELL_CRW_D_GT3,		"Haswell CRW Desktop (GT3)" },
+	{PCI_CHIP_HASWELL_CRW_D_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_M_GT1,		"Haswell CRW Mobile (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_M_GT2,		"Haswell CRW Mobile (GT2)" },
-	{PCI_CHIP_HASWELL_CRW_M_GT3,		"Haswell CRW Mobile (GT3)" },
+	{PCI_CHIP_HASWELL_CRW_M_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_S_GT1,		"Haswell CRW Server (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_S_GT2,		"Haswell CRW Server (GT2)" },
-	{PCI_CHIP_HASWELL_CRW_S_GT3,		"Haswell CRW Server (GT3)" },
+	{PCI_CHIP_HASWELL_CRW_S_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_B_GT1,		"Haswell CRW (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_B_GT2,		"Haswell CRW (GT2)" },
-	{PCI_CHIP_HASWELL_CRW_B_GT3,		"Haswell CRW (GT3)" },
+	{PCI_CHIP_HASWELL_CRW_B_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_E_GT1,		"Haswell CRW (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_E_GT2,		"Haswell CRW (GT2)" },
-	{PCI_CHIP_HASWELL_CRW_E_GT3,		"Haswell CRW (GT3)" },
+	{PCI_CHIP_HASWELL_CRW_E_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_VALLEYVIEW_PO,		"ValleyView PO board" },
 	{-1,					NULL}
 };
